docs/rules-engine/operators.md

```diff
@@ -71,6 +72,8 @@ Example of usage :
 | All[]       | lengthGreaterThanOrEquals | number of ≥            | Check if the number of values of the variable is greater or equal to a specific value |
 | All[]       | lengthGreaterThan         | number of >            | Check if the number of values of the variable is greater than a specific value |
 
+> **Note**: For the operators comparing a text variable to a pattern, the special characters used in the patterns should contain a double backslash (`\\`).
+> For example, to check if a string contains a `/`, the pattern would need to include `\\/`.
```

Comment:
You should also specify that you support `/^myRegExp.*$/i`(ES RegExp) and `^myregexp.*$` (regexp content).
(also, with my suggestion, the `\\/` is not mandatory, it is already not mandatory in the *regexp content* format)